test/__tests__/user_transformation.test.js

```diff
@@ -1073,6 +1073,7 @@ describe("User transformation", () => {
       trRevCode.versionId,
       [libraryVersionId],
       trRevCode,
+      null,
```

Comment:
No wrong in passing null as a test. ideally default value should be [].
No change required here